### PR TITLE
Updating fluent-bit version and installing in CS for local debugging

### PR DESCRIPTION
**Pull request by aflinchb**
## Purpose of PR

- Update fluent-bit to use latest version
- Install fluent-bit in CS for local debugging

## Type of PR

- [x] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Issues Closed

- Closes https://github.com/retaildevcrews/akdc/issues/195
- Closes https://github.com/retaildevcrews/akdc/issues/196

Waiting to see if fluent-bit issue resolves with new tag
https://github.com/fluent/fluent-bit/issues/5254

### CODE REVIEW

File: deploy/bootstrap/fluentbit/deployment.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ spec:
           type: RuntimeDefault
       containers:
         - name: fluent-bit
-          image: fluent/fluent-bit:1.5
+          image: fluent/fluent-bit:1.9.0
```
[the remark this answers is not in the document]
Comment (aflinchb) [reply deploy/bootstrap/fluentbit/deployment.yaml:28]: Good catch - I see the latest available is 1.9.3.  For this release they dont have a 1.9 tag, I'm reaching out to see if there are plans to do so or if there is another tag that could be used.

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -6,7 +6,19 @@ FROM ghcr.io/cse-labs/k3d:latest
...
 ARG USERNAME=vscode
 
-# log the docker build start / complete events
-RUN echo "$(date)    docker build start" >> /home/$USERNAME/status && \
-    echo "$(date)    docker build complete" >> /home/$USERNAME/status && \
-    chown -R $USERNAME:$USERNAME /home/$USERNAME
+# log the docker build start event
+RUN echo "$(date +'%Y-%m-%d %H:%M:%S')    docker build start" >> /home/$USERNAME/status
+
+# install fluent-bit
```
Comment (bartr) [inline .devcontainer/Dockerfile:12]: fluent bit is now installed in the upstream DinD image, so you can delete this

File: deploy/bootstrap/fluentbit/deployment.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (aflinchb) [reply deploy/bootstrap/fluentbit/deployment.yaml:28]: https://github.com/fluent/fluent-bit/issues/5254

[the remark this answers is not in the document]
Comment (aflinchb) [reply deploy/bootstrap/fluentbit/deployment.yaml]: This is resolved now, and the yaml has been updated to use 1.9. 

https://github.com/fluent/fluent-bit/issues/5254

**Pull request merged**